index.html

```diff
@@ -2223,7 +2223,7 @@ <h3>The <dfn>NDEFScanOptions</dfn> dictionary</h3>
       </pre>
       <p>
         The <dfn>signal</dfn> property allows to abort the
-        [=NDEFReader/scan()=] operation.
+      {{NDEFReader/scan()}} operation.
```

Comment:
@kenchris Nit: some spaces have been removed.

Reply:
True, shouldn't change rendering though, but would be nice to fix